src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -471,6 +471,14 @@ def describe_service(self, service_type: Optional[str] = None, service_name: Opt
         """
         raise NotImplementedError()
 
+    def describe_prometheus_service_spec(self):
```

Comment:
do we need to expose this? there is already a `describe_service` which is used for describing all the services. A separate interface prometheus might not be needed

Reply:
?

Reply:
Functions only need to be added to _interface.py if they are going to be getting called by something in orchestrator/module.py that needs to route it to the backend orchestrator (cephadm, typically). Afaict, `describe_prometheus_service_spec` is only being called from within the cephadm module, so this definition of it in _interface.py shouldn't be necessary.